src/BlockInfo.cpp

```diff
@@ -313,6 +313,7 @@ cBlockInfo::cBlockInfoArray::cBlockInfoArray()
 	Info[E_BLOCK_SANDSTONE_STAIRS             ].m_Transparent = true;
 	Info[E_BLOCK_SAPLING                      ].m_Transparent = true;
 	Info[E_BLOCK_SIGN_POST                    ].m_Transparent = true;
+	Info[E_BLOCK_SNOW                         ].m_Transparent = true;
```

Comment:
I'm confused as to why you have added this - a full block of snow is not transparent.